### ISSUE TITLE
Lore Friendly Version for FiveM?

### ISSUE THREAD

**Issue by starfiregaming75** [Motogege91/moto_greeneats#2]
FiveM recently made it so you can't use real branding is their anyway to swap out the real brands for fake brands in the store so it doesn't violate FiveM's new terms? I love the little grocery store. It's my favorite thing and I really wanted to add it to my server.

**Comment by Motogege91** [comment]
Yes, there's the .ytd with all the textures inside. Actually, there are 3-4 brands, like bottles and some bridge stuffs, but i'm working on a V2 with all fake generated stuffs

**Comment by starfiregaming75** [comment]
Thank you so so much, I love your little grocery store, I appreciate you taking time to make it with fake brands so it can be used :)
